docs/explanation/configs.rst

```diff
@@ -29,30 +35,30 @@ value resolution is as follows:
 Configs with Checkbox Remote
 ============================
 
-When the Checkbox Slave starts, it looks for config files in the same places
-that local Checkbox session would look (on the Slave side).
-If the Master uses a Launcher, then the values from that Launcher take
-precedence over the values from configs on the Slave side.
+When the :term:`Checkbox Agent` starts, it looks for config files in the same
+places that local Checkbox session would look (on the :term:`DUT` side). If
```

Comment:
later on you use the phrase "run on the agent". Any reason why it's DUT here?

Reply:
I wanted to let users know Agents and DUT are interchangeable names, but I've replaced it with Agent as you suggest.

Fixed in [c0681c6](https://github.com/canonical/checkbox/pull/562/commits/c0681c6b3ef6bc1d79d3a05f4269e64a65800ec6)